jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -683,6 +748,33 @@ YUI.add('user-profile', function() {
     },
 
     /**
+      Generate the header for the budgets.
+
+      @method _generateBudgetHeader
+      @returns {Array} The markup for the header.
+    */
+    _generateBudgetHeader: function() {
```

Comment:
I'm sure we can come up with some table components in the future to simplify all of this faux table stuff.